Makefile

```diff
@@ -117,9 +117,21 @@ rpm-linux: build-linux ## Build the linux binary and package it as a .rpm for Re
 	rpm-build/build-rpm.sh $(package_binary) $(DOCKER_IMAGE_RPM) $(PACKAGE_PATH) $(CONTROLLER_BASE_URL) $(VERSION)
 	rm -f $(package_binary)	
 
+
+.PHONY: build-docs
+build-docs:
+        # make docs md file
```

Comment:
I'm not sure how picky `make` is on spacing, but I notice this comment line has all spaces, and the following lines have two tabs. Other scripts in the Makefile only have one tab. Let's try to be consistent.